resources/user-agents/browsers/openwave-mobile-browser/openwave-mobile-browser-4-0.json

```diff
@@ -28,6 +28,10 @@
           "match": "UPG1* UP.Browser/#MAJORVER#.#MINORVER#*"
         },
         {
+          "match": "Nokia* UP.Browser/#MAJORVER#.#MINORVER#*",
```

Comment:
When I compared my changes for this PR against current master, I noticed that the addition of the `Nokia*/*` pattern in `nokia-browser.json` was causing some agents to be detected as `Nokia Browser` instead of `UC Browser` so I added these to keep that from happening.

Of course, the devices themselves could be added to these files where appropriate, but that would require having a huge list of UC Browser patterns that start with "Nokia*" and is still not a guarantee that all would be covered.